### ISSUE TITLE
Harvest a raster layer from ArcGIS Server MapServer/ImageServer?

### ISSUE THREAD

**Issue by marcgr** [Esri/geoportal-server#288]
I can successfully register a network resource pointed at a MapServer or ImageServer endpoint, but when I approve it and have it harvest, it will only pull out the vector layers, not the raster layers.  So for the ImageServer, it doesn't pull anything though it shows success.   Is this supported, or am I missing something?

**Comment by jsta** [comment]
I am also interested in this functionality.

**Comment by mhogeweg** [comment]
correct, we only harvest the service endpoints in the harvester. I have published this [script](https://gist.github.com/mhogeweg/95717823010fd5149b9be7312e4454ff) that loops through the image server and generates an FGDC XML document for every image. It will create keywords for the various image attributes.

This is a bit of a hack and you could also push JSON directly to Geoportal Server 2.x.

After this you can configure additional search facets in the search page.
